src/main/java/org/truffleruby/core/symbol/SymbolNodes.java

```diff
@@ -84,6 +85,8 @@ public long hash(DynamicObject symbol) {
     @CoreMethod(names = "to_proc")
     public abstract static class ToProcNode extends CoreMethodArrayArgumentsNode {
 
+        private final FrameDescriptor descriptor = new FrameDescriptor();
```

Comment:
It would be nice to allocate only of these empty FrameDescriptor that is never added slots.